lib/pacemaker/pcmk_output.c

```diff
@@ -1917,6 +1917,11 @@ pcmk__cluster_status_text(pcmk__output_t *out, va_list args)
         }
     }
 
+    /* Always print verification errors if there are any, even if not requested */
+    if (pcmk_is_set(section_opts, pcmk_section_validate)) {
+        rc = pcmk__verify(scheduler, out, scheduler->input);
+    }
```

Comment:
This is a good step to ensure it's working as desired. However for the final output, I don't want to clutter the status display with what may be a long list of problems, so what we can do is create a temporary output object with output type "none", call `pcmk__verify()` regardless of whether the flag is set or not, then use only the return code to display a message in the status. (That will also make it easy to always show it when there's a problem.)

Basically if there's no error, it should say something to the effect that there are no CIB validation issues, and if there is an error, it should say there are CIB validation issues and to run crm_verify for more details.